src/deadline/job_attachments/api/attachment.py

```diff
@@ -90,6 +90,7 @@ def attachment_upload(
     root_dirs: List[str] = [],
     path_mapping_rules: Optional[str] = None,
     upload_manifest_path: Optional[str] = None,
+    manifest_name_suffix: str = "input",
```

Comment:
nit - what do you think if we just had "upload file name", and the customer or caller can provide the full name? 

Or better yet, what if we just uploaded it with the same name as the manifest file? Users can rename files before running the API or CLI

Reply:
Yeah, wondering why do we need/want the suffix? What happens if a customer provides an empty suffix?

Reply:
I don't see if `_input` suffix is using anywhere, that's probably for visibility only. However, for `_output`, it's used for download. Without the `_output` suffix, job download cannot find the manifests and download corresponding files.

https://github.com/godobyte/deadline-cloud/blob/mainline/src/deadline/job_attachments/download.py#L174-L175

Reply:
But that's the question - feature parity for the heck of it is not the goal of creating the API. 

The name of the file on S3 simply by naming / moving the file and using it. Adding a new parameter to accomplish this seems odd.

If we want to have a parameter, I would rather make it generic like "Upload file name", which lets the caller customize the entire name instead of just the postfix.

Reply:
See here: https://github.com/aws-deadline/deadline-cloud/blob/43cd3623f3bb6cd2c0295205fc61f5677975dd51/src/deadline/job_attachments/upload.py#L231

All the `postfix` does is a string format. For the CLI and API we should just have the input as the right name to begin with.

Reply:
My concern with being able to upload any customer-defined manifest file names are 
1. Violating the file name hash to root mapping Job Attachment relies on to get upload source and download destination.
2.  Breaking the existing logic we have for [output download that relies on file suffix](https://github.com/godobyte/deadline-cloud/blob/mainline/src/deadline/job_attachments/download.py#L195).

If we allow any name for manifests, the above 2 cases need to be proper thought out with error handling.